addons/spreadsheet/static/src/global_filters/plugins/global_filters_core_view_plugin.js

```diff
@@ -550,6 +556,14 @@ export class GlobalFiltersCoreViewPlugin extends OdooCoreViewPlugin {
         return new Domain([[field, "in", [toBoolean(value[0]), toBoolean(value[1])]]]);
     }
 
+    _getNumericDomain(filter, fieldMatching) {
+        const value = this.getGlobalFilterValue(filter.id);
+        if (!value || !fieldMatching.chain) {
```

Comment:
`value` could be 0, thus you need to explicitly check that value is not `undefined`